test/interval_tests/complex.jl

```diff
@@ -14,7 +14,6 @@ end
 
 @testset "Complex functions" begin
     Z = (3 ± 1e-7) + (4 ± 1e-7)*im
-    @test sin(Z) == Interval(3.853734949309744, 3.8537411265295543) - Interval(27.016810169394066, 27.016816346613904)*im
-
+    @test sin(Z) == complex(sin(real(Z))*cosh(imag(Z)),sinh(imag(Z))*cos(real(Z)))
```

Comment:
Nice!

Reply:
Thanks! I though this way is more stable :-)